fiftyone/core/session.py

```diff
@@ -204,6 +209,9 @@ class Session(foc.HasClient):
         terminate the session.
 
     Args:
+        color_pool (None): a list of browser supported color strings from
```

Comment:
In the future I think the number of App-config-related parameters will get a bit unwieldy. How about encapsulating `color_pool, height, show_attributes, show_confidence` in a new `AppConfig` class that can be passed to `Session`?

By default, `AppConfig` would inherit default values from `fo.config` for any `None`-valued properties. Example usage:

```py
config = fo.AppConfig()  # default values from `fo.config`
config.show_attributes = False  # customize

session = fo.launch_app(dataset, config=config)

session.config.show_attributes = True  # let's change it back
```